L65-python-package-name.md

```diff
@@ -0,0 +1,132 @@
+Additional PyPI Packages for gRPC Python
+----
+* Author(s): lidiz
+* Approver: gnossen
+* Status: Draft
+* Implemented in: python
+* Last updated: 05/13/2020
+* Discussion at: **TBD**
+
+## Abstract
+
+gRPC Python is uploaded as `grpcio` on PyPI, but there is another package named
+`grpc`. Hence some users are confused. This document proposes gRPC Python to
+upload additional packages named `grpc` and `grpc-*` as proxies to our official
+packages.
+
+## Background
+
+### Package `grpc`
+
+Package `grpc` is an alternative version of gRPC Python only supports Python 2,
```

Comment:
We probably want to add a bit of history as well on how we obtained ownership of the package name.

Reply:
Add a link to https://github.com/pypa/pypi-support/issues/3.